### PR TITLE
:tada:  feat(tenable/alsid): migrate format :tada: 

### PR DESCRIPTION
**Pull request by PierrickV**
### Changelog

| Before the update  | After the update |
| ------------- | ------------- |
| (no impact expected) at the parser level `action.properties` is of type list|`action.properties` is of type dict|
| `action.properties.OperatingSystem: 'Windows'`  | `action.properties.OperatingSystem: 'Windows Server 2012 R2 Standard'`   |
| `action.properties.OperatingSystemVersion`: '6.3' | `action.properties.OperatingSystemVersion: '6.3 (9600)'`   |
| `action.properties.DisplayName`: DSC | `action.properties.DisplayName: DSC UCN Export` |
| broken value in `action.properties.DangerousAceList` | fixed, the field now contains the complete value |
|broken value in `action.properties.DistinguishedName`|fixed, the field now contains the complete value|
|`action.properties.ADdevianceID` type float|`action.properties.ADdevianceID` type int|
|`action.id` type float| `action.id` type int|
|`action.properties.alsidAttributeValue: ["Microsoft.MicrosoftEdge_8wekyb3d8bbwe"]`|`action.properties.alsidAttributeValue.0: Microsoft.MicrosoftEdge_8wekyb3d8bbwe`|
| Event starting by 2 not parsed|Event starting by 2 parsed|

### CODE REVIEW

**Comment by PierrickV** [comment]
@SEKOIA-IO/sekoia-io-integration-team @squioc : Ready for review :pray:

**Review by TOUFIKIzakarya — COMMENTED** [review]
Hello @PierrickV. thanks for the work.
 I have some remarks regarding taxonomy, I think that should go with [this](https://github.com/SEKOIA-IO/intake-formats/blob/main/doc/taxonomy.md). 
 Also can't see logos, smart desc and Changelog file ? is it an exception for this parser ?

**Comment by PierrickV** [comment]
@TOUFIKIzakarya:
- fields: it is a migration, we need the result to be ISO with the previous version
- logo: https://github.com/SEKOIA-IO/intake-formats/blob/main/Tenable/alsid/_meta/logo.png
- smart desc: I am just migrating the parser's code, not adding content
- changelog: I will add one :+1:

**Comment by PierrickV** [comment]
@TOUFIKIzakarya I updated the changelog :smile:

File: Tenable/alsid/tests/alert_certificate_DANG_ACCESS.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      "outcome": "success",
+      "outcome_reason": "R-CERTIF-TEMPLATE-ACL",
+      "properties": {
+        "ADdevianceID": 1996839,
+        "ADdomainName": "ad.domain",
+        "ADforestName": "ad.domain",
+        "ADobject": "CN=DSCUCNExport,CN=Certificate Templates,CN=Public Key Services,CN=Services,CN=Configuration,DC=ad,DC=domain",
+        "alertID": 1,
+        "alertSeverityLevel": "critical",
+        "eventID": "76485473\"       \"DistinguishedName\"=\"CN=DSCUCNExport,CN=Certificate Templates,CN=Public Key Services,CN=Services,CN=Configuration,DC=ad,DC=domain\"                                                                                                              \"DangerousAceList\"=\"[{\"Item1\":\"A;;LCRPWPRCWDWO;;;S-1-5-21-1229472208-2678311744-2345022811-345849\",\"Item2\":\"S-1-5-21-1229472208-2678311744-2345022811-345849\",\"Item3\":\"urdom.ad.domain\\\\AC750-DSI-FOO-Espace de                                travail-GSW-Adm\",\"Item4\":[{\"Item1\":\"Modify permissions\",\"Item2\":\"\"},{\"Item1\":\"Modify owner\",\"Item2\":\"\"},{\"Item1\":\"Write all properties\",\"Item2\":\"\"}]}]"
```
Comment (TOUFIKIzakarya) [inline Tenable/alsid/tests/alert_certificate_DANG_ACCESS.json:21]: Is the eventID "76485473" or the whole line ?

**Comment by Sengthay** [comment]
Thanks @PierrickV
According to the changelog, the migration should have minimal impact.
We should be able to deploy fast and communicate a simple changelog

**Comment by PierrickV** [comment]
ok with detection (cc @r1chev)

File: Tenable/alsid/tests/alert_certificate_DANG_ACCESS.json (hunk 1 above)
Comment (PierrickV) [reply Tenable/alsid/tests/alert_certificate_DANG_ACCESS.json]: @TOUFIKIzakarya  Good catch thank you!

It should now be fixed

**Review by TOUFIKIzakarya — DISMISSED** [review]
LGFM

**Review by squioc — DISMISSED** [review]
The parser is ok. However, the format lack of smart-descriptions

**Comment by PierrickV** [comment]
Thank you for your review @squioc  :smile: 

Would you like an issue so that me remember to add the missing smart descriptions ?

My goal here is to focus on parser code so I wont add smart descriptions or improve the parser during the migration.

**Comment by PierrickV** [comment]
:rocket: I think that we are ready for this one @Sengthay

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**